src/cpp/cpp_typecheck_bases.cpp

```diff
@@ -141,19 +141,18 @@ void cpp_typecheckt::add_base_components(
     vbases.insert(from_name);
 
   // look at the the parents of the base type
-  forall_irep(it, from.find(ID_bases).get_sub())
+  for(const auto &b : from.bases())
   {
-    irep_idt sub_access=it->get(ID_access);
+    irep_idt sub_access = b.get(ID_access);
```

Comment:
Isn't there a `get_access()`?

Reply:
Not for a base (class), only for components.